index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Dosis:wght@500&family=Secular+One&display=swap" rel="stylesheet">
+    <link rel="shortcut icon" type="image/jpg" href="Nerd Emoji With Glasses.png"/>
+    <link href= "normalize.css" type="text/css" rel="stylesheet"/>
+    <meta name="viewport" content="width=device-width">
+    <script src="index.js" defer></script>
+    <link href="styles.css" rel="stylesheet" type="text/css" />
+    <title>Word Nerd</title>
+</head>
+<body>
+    <nav>
+        <button id="instructions">?</button>
+        <div class="overlay">
+            <div class="modal">
+                <button class="closeBtn">&times;</button>
+                <h1>Instructions for W&#129299RD NERD</h1>
+                <ul>
+                    <li>Guess the 5 letter word!</li>
+                    <li>You have 6 attempts to guess the word correctly.</li>
+                    <li>Enter your guess from the virtual keypad or your keyboard letters, then press 'submit'.</li>
+                    <li>Each guess must be 5 letters long, with no numbers or spaces.</li>
+                    <li>Each tile will change colour to reflect the accuracy of your guess.</li>
+                    <li><p class="correct">Green</p> = The letter is in the right place</li>
+                    <li><p class="rightLetter">Yellow</p> = The letter is in the word, but not in the right place</li>
+                    <li><p class="wrongLetter">Dark grey</p> = The letter is not in the word</li>
+                    <li>After the game has finished, you can play again by clicking the 'Replay' button</li>
+                    <li>Have fun W&#129299RD NERD;!</li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <main>
+        <div class="title"><h1>W&#129299RD NERD</h1></div>
+        <div class="letterTiles">
+
+        </div>
+        <div class="wordInput Input Outcome">
+            <form method="post" action="">
+                <input type="text" placeholder="Guess Word ..." class="textInput">
+                <input type="submit" value="Enter" class="enter submitFormButton">
+                <input type="submit" value="Retry" class="retry retryButton">
+            </form>
+        </div>
+        <section class="keypadContainer">
+            <div class="keypadRow">
+                <button class="keypadKey" data-letter="Q">Q</button>
```

Comment:
might be worth making all the data-letters lower case to match the data you will be working with

Reply:
Done